### PR TITLE
📊 UN WPP population five year age-groups for 0-100, instead of 0-20

### PR DESCRIPTION
**Pull request by spoonerf**
* Adding 5-year age-groups for population across the whole age range (0-100) - for [this chart](http://staging-site-five-year-pops/admin/charts/7691/edit)
* Removing the new 5-year age-groups from the explorer step, so the explorer is unchanged
* Adding an experimental dataset to show the populous age-group for each country-year - for[ this chart](http://staging-site-five-year-pops/admin/charts/7690/edit)

### CODE REVIEW

File: etl/steps/data/explorers/un/2022/un_wpp.py
Comment (lucasrodes) [inline etl/steps/data/explorers/un/2022/un_wpp.py:32]: very minor, I generally prefer using `~` than `-` as the negation operator. I think is better practice, but I just can't remember why : D

File: etl/steps/data/garden/un/2024-03-14/un_wpp_most.meta.yml
Comment (lucasrodes) [inline etl/steps/data/garden/un/2024-03-14/un_wpp_most.meta.yml:8]: I find it a bit redundant to use "age" twice in the sentence.

Maybe something like

> Five-year age group with the highest population.

maybe that's too simple?

Another example:

> The largest five-year population age group.

**Review by lucasrodes — APPROVED** [review]
LGTM! Nice that you are working on these charts.

I was wondering, [this chart](http://staging-site-five-year-pops/admin/charts/7690/edit) would really benefit from having brackets instead of individual boxes. Maybe we could have that for ordinal indicators, like this one.

File: etl/steps/data/explorers/un/2022/un_wpp.py
Comment (spoonerf) [reply etl/steps/data/explorers/un/2022/un_wpp.py:32]: Ah okay - I will update it to that!

File: etl/steps/data/garden/un/2024-03-14/un_wpp_most.meta.yml
Comment (spoonerf) [reply etl/steps/data/garden/un/2024-03-14/un_wpp_most.meta.yml:8]: Good point - yours is much better!

**Comment by spoonerf** [comment]
Thanks @lucasrodes! 

Because the UN WPP dataset has so many downstream dependencies should I wait until this evening to merge it? Also is it worth waiting for the ordinal map brackets to be a feature before merging this?

**Comment by lucasrodes** [comment]
> Because the UN WPP dataset has so many downstream dependencies, should I wait until this evening to merge it?
Good call! Would be nice to have a scheduler.

> Also is it worth waiting for the ordinal map brackets to be a feature before merging this?

Nah! Let's take a note and improve this once that's out.

**Pull request merged**
